### DISCUSSION TITLE
Error message when editing proband relationship

### DISCUSSION THREAD

**Discussion by bnouyou** [broadinstitute/seqr#1996] [Q&A] [answered]
Hello,
I have pulled the last docker image and done the migration of the database.
I see that some families haven't the pedigree image anymore.
I wanted to set back the information about proband relationship in "Edit Families & Individuals" but I have this error message when I click on "Apply" :
![image](https://user-images.githubusercontent.com/22322228/127154709-2440bd9f-b145-4be6-8cb7-5a27c76bcd5e.png)

This user is "superuser", so I suppose he should have the good rights.

I have noticed that there is already an issue for this problem (#1659) and it has been closed. It's explained to remove the column "proband relationship" in the file. But I don't have this column in my file.
Moreover, I can't change the proband relationhip on the form directly because of the error message.

Thank you in advance for your help.

**Comment by hanars** [comment, marked as answer]
proband relationship is an internal-only field we use for Broad seqr's CMG reporting requirements. It is not meant to be available in local installations. If you *really* want to use it, you need to create a user group, set `ANALYST_USER_GROUP` to the name of that group in the docker compose environment, and then make sure to add your user to that group. But again, we don't really recommend this for local installations as the features this enables are not broadly helpful

**Reply by bnouyou** [reply]
Ok I understand.
I don't really want to use this field, actually I just want to re-create the images of pedigrees. And I don't know how to do.

**Reply by hanars** [reply]
the easiest to regenerate the images is to edit the sex or affected status of an individual, and then change it back

**Reply by bnouyou** [reply]
Great, it's working. Thank you !
